hugr-core/src/export.rs

```diff
@@ -371,22 +361,29 @@ impl<'a> Context<'a> {
                 let mut args = BumpVec::new_in(self.bump);
                 args.extend(call.type_args.iter().map(|arg| self.export_type_arg(arg)));
                 let args = args.into_bump_slice();
+                let func = self.make_term(model::Term::Apply(symbol, args));
+
+                // PERFORMANCE: Avoid exporting the signature here again.
```

Comment:
Add a TODO tag too, so it's still highlighted/searchable?
```suggestion
                // TODO PERFORMANCE: Avoid exporting the signature here again.
```